src/Sentry.Unity/Integrations/UnityApplicationLoggingIntegration.cs

```diff
@@ -90,6 +94,15 @@ internal void OnLogMessageReceived(string? condition, string? stackTrace, LogTyp
             _hub.AddBreadcrumb(message: condition, category: "unity.logger", level: ToBreadcrumbLevel(type));
         }
 
+        private static StackTraceLogType GetStackTraceLogType()
+        {
+#if UNITY_EDITOR
+            return PlayerSettings.GetStackTraceLogType(logType);
+#else
+            return StackTraceLogType.ScriptOnly;
```

Comment:
Couldn't it be Full outside of the editor?

Reply:
Oh I didn't know that `Application.GetStackTraceLogType` exists. I've changed it.